### PR TITLE
Blaine was here

### PR DESCRIPTION
**Pull request by Blakeinstein**

### CODE REVIEW

**Comment by Blakeinstein** [comment]
Hope I made it in time! Looks rad.

**Review by shajanjp — CHANGES_REQUESTED** [review]
Change `Blakeinstein.github.io` to `https://blakeinstein.github.io`

**Comment by Blakeinstein** [comment]
rebased and added said change!

**Comment by Blakeinstein** [comment]
confused as to why this check failed.

File: domains/blaine.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "The face of Blaine, on the web",
+    "repo": "https://github.com/Blakeinstein/Blakeinstein.github.io",
+    "owner": {
+      "username": "Blakeinstein",
+      "email": "rishi1998@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://blakeinstein.github.io",
```
Comment (phenax) [inline domains/blaine.json:9]: CNAME has to be a host. Full URLs dont work
```suggestion
      "CNAME": "blakeinstein.github.io",
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Review by phenax — APPROVED** [review]

**Pull request merged**